index.html

```diff
@@ -0,0 +1,47 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1>Hello World, we are software developers!
+    </h1>
+    <main id="main">
+      <h1 id="title">Nelson Mandela</h1>
```

Comment:
Heading Tag Should be Semantic  Order. give  Please ensure the title of page can have <h1> tag rest have tag other than h1 in semantic order

Reply:
- [ ] Kindly rewrite this comment, it's not easily understood, you can visit [here](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) to learn more about semantics and be able to give your observation clearly